src/Dialogs/Plane/PlaneListDialog.cpp

```diff
@@ -150,12 +151,20 @@ PlaneListWidget::CreateButtons(WidgetDialog &dialog)
 }
 
 void
-PlaneListWidget::Prepare(ContainerWindow &parent, const PixelRect &rc) noexcept
+PlaneListWidget::Prepare(ContainerWindow& parent, const PixelRect& rc) noexcept
```

Comment:
Why hide this (bad) coding style change in this commit?

Reply:
:( I have to check my editor